xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserSheet.xml

```diff
@@ -448,17 +448,23 @@ return XWiki;
   width: 13em;
 }
 
-#avatar p {
-  text-align: center;
-}
+#avatar {
+  p {
+    text-align: center;
+  }
+  img{
```

Comment:
We should add a space here to keep consistency in the codestyle:

> put a space between selector and declaration start, ex. "a {}"

from https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/